app/models/miq_request.rb

```diff
@@ -5,6 +5,7 @@ class MiqRequest < ApplicationRecord
   belongs_to :destination,       :polymorphic => true
   belongs_to :requester,         :class_name  => "User"
   belongs_to :tenant
+  belongs_to :service_order
```

Comment:
@gmcculloug do we have to worry about prior MiqRequests that aren't in a ServiceOrder?